controllers/warp/warp.go

```diff
@@ -67,18 +71,56 @@ func (w *Watcher) Run(ctx context.Context) error {
 	}
 
 	for _, source := range w.configuration.Sources {
-		if strings.HasPrefix(source.Path, "/dogu") {
+		if strings.HasPrefix(source.Path, "/dogu") || strings.HasPrefix(source.Path, "dogu") {
 			w.startVersionRegistryWatch(ctx, source.Path)
 			continue
 		}
 
-		// TODO config/_global
-
-		// TODO config/
+		if strings.HasPrefix(source.Path, "/config/_global") || strings.HasPrefix(source.Path, "config/_global") {
+			w.startGlobalConfigWatch(ctx, source.Path)
+			continue
+		}
 	}
 
 	return nil
+}
 
+func (w *Watcher) startGlobalConfigWatch(ctx context.Context, sourcePath string) {
+	ctrl.LoggerFrom(ctx).Info(fmt.Sprintf("start global config watcher for source [%s]", sourcePath))
+	filter := libconfig.KeyFilter(libconfig.Key(sourcePath))
```

Comment:
The watch doesn't work for external-Links. This might be due to the fact that it is a directory and a directoryFilter would be needed here?